style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Inter", sans-serif;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 15px;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+}
+
+header {
```

Comment:
[Optional] Using _header_ in your CSS is good because header tag is a requirement to be used as a container, but in your _index.html_, you have not used the header tag as a container. I'll recommend you look into that.